tst/testinstall/meatauto.tst

```diff
@@ -20,7 +20,7 @@ rec( dim := 3, failed := false, field := GF(2), index := [  ],
   mat := [ <a GF2 vector of length 3>, <a GF2 vector of length 3> ], 
   vec := [ 0*Z(2), Z(2)^0 ], weights := [ 1, 2 ] )
 gap> SMTX_NullspaceEqns(e);
-[ [ Z(2)^0, 0*Z(2), Z(2)^0 ] ]
+[ <a GF2 vector of length 3> ]
```

Comment:
Is it possible to see the "GF2 vector of length 3"? Since the tests worked, they are most probably the same.

Reply:
Good point, I've added a test to check the return value.

Reply:
Great, thanks!